apps/docs/src/app/platform/component-docs/platform-list/platform-list-examples/platform-list-with-i18n-example.component.html

```diff
@@ -0,0 +1,24 @@
+<h5> Declarative Approach</h5>
+<fdp-list>
+    <fdp-standard-list-item i18n-title title="Title1">
+    </fdp-standard-list-item>
+
+    <br />
+    <br />
+    <h3>After translation with help of i18n</h3>
+
+    <fdp-standard-list-item title="objet 1">
+    </fdp-standard-list-item>
+
+    <br />
+    <br />
+    <h3>Transalation of footer text in applicaition with help of i18n</h3>
```

Comment:
i am not sure if this is the right way to show this example

Reply:
list and list item does not have static text, as discussed in OPS, we are just trying to show how i18n can be used in list and its items.

Reply:
understood, but still, giving a different language here may not be needed. We just need to show that i18n-title etc is the way to use it. I think other components are also doing only that.

Reply:
got your point, I added it for a sample as per discussion,  in fact, I was thinking, having i18n without static text, won't help much. Let's discuss this once again in OPS as now we have a example.

Reply:
As discussed with Manav and team, we will be having only in special cases, so removing it.